app/views/user_sessions/_form.html.erb

```diff
@@ -25,9 +25,12 @@
       
       <label for="username"><%= t('user_sessions.new.username') %></label>
       <%= f.text_field :username, { tabindex: 1, placeholder: "Username", class: 'form-control', id: 'username-login', required: true } %>
-      <label for="password"><%= t('user_sessions.new.password') %></label>
-      <%= f.password_field :password, { tabindex: 2,placeholder: "Password",class: 'form-control', id: 'password-signup', onpaste: 'return false;' , required: true } %>
-  
+      <div class="form-group has-feedback">
+      <label class="control-label" for="password"><%= t('user_sessions.new.password') %></label>
+        <%= f.password_field :password, { tabindex: 2,placeholder: "Password",class: 'form-control', id: 'password-signup', onpaste: 'return false;' , required: true } %>
+  <a id="visibility" class="glyphicon glyphicon-eye-open form-control-feedback" aria-hidden="true" style="color:#888;pointer-events: auto;"></a>
```

Comment:
This is great -- could we use `fa fa-eye` instead since we use mostly FontAwesome icons in this project? Thank you!

Reply:
I'll update it @jywarren